src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -14,109 +14,158 @@
+  private static final String HDFS_NAMENODE_PRINCIPAL_DOC = "The principal for HDFS Namenode.";
+  public static final String HDFS_NAMENODE_PRINCIPAL_DEFAULT = "";
+  private static final String HDFS_NAMENODE_PRINCIPAL_DISPLAY = "HDFS NameNode Kerberos Principal";
+
+  public static final String KERBEROS_TICKET_RENEW_PERIOD_MS_CONFIG = "kerberos.ticket.renew.period.ms";
+  private static final String KERBEROS_TICKET_RENEW_PERIOD_MS_DOC =
+      "The period in milliseconds to renew the Kerberos ticket.";
+  public static final long KERBEROS_TICKET_RENEW_PERIOD_MS_DEFAULT = 60000 * 60;
+  private static final String KERBEROS_TICKET_RENEW_PERIOD_MS_DISPLAY = "Kerberos Ticket Renew Period in Milliseconds";
+
+  // Connector group
+  public static final String FLUSH_SIZE_CONFIG = "flush.size";
+  private static final String FLUSH_SIZE_DOC =
+      "Number of records written to HDFS before invoking file commits.";
+  private static final String FLUSH_SIZE_DISPLAY = "Flush Size";
+
+  public static final String ROTATE_INTERVAL_MS_CONFIG = "rotate.interval.ms";
+  private static final String ROTATE_INTERVAL_MS_DOC =
+      "The time interval in milliseconds to invoke file commits. This configuration ensures that "
+      + "file commits are invoked every configured interval. This configuration is useful when data "
+      + "ingestion rate is low and the connector didn't write enough messages to commit files."
+      + "The default value -1 means that this feature is disabled.";
+  private static final long ROTATE_INTERVAL_MS_DEFAULT = -1L;
+  private static final String ROTATE_INTERVAL_MS_DISPLAY = "Rotate Interval in Milliseconds";
+
+  public static final String RETRY_BACKOFF_CONFIG = "retry.backoff.ms";
+  private static final String RETRY_BACKOFF_DOC =
+      "The retry backoff in milliseconds. This config is used to "
+      + "notify Kafka connect to retry delivering a message batch or performing recovery in case "
+      + "of transient exceptions.";
+  public static final long RETRY_BACKOFF_DEFAULT = 5000L;
+  private static final String RETRY_BACKOFF_DISPLAY = "Retry Backoff in Milliseconds";
 
   public static final String SHUTDOWN_TIMEOUT_CONFIG = "shutdown.timeout.ms";
   private static final String SHUTDOWN_TIMEOUT_DOC =
       "Clean shutdown timeout. This makes sure that asynchronous Hive metastore updates are "
       + "completed during connector shutdown.";
   private static final long SHUTDOWN_TIMEOUT_DEFAULT = 3000L;
-
-  public static final String SCHEMA_COMPATIBILITY_CONFIG = "schema.compatibility";
-  private static final String SCHEMA_COMPATIBILITY_DOC =
-      "The schema compatibility rule to use when the connector is observing schema changes. The "
-      + "supported configurations are NONE, BACKWARD, FORWARD and FULL.";
-  private static final String SCHEMA_COMPATIBILITY_DEFAULT = "NONE";
+  private static final String SHUTDOWN_TIMEOUT_DISPLAY = "Shutdown Timeout in Milliseconds";
```

Comment:
Some of these with units might make more sense to list as `"Shutdown Timeout (ms)"` instead.